app/views/gobierto_indicators/indicators/_year_breadcrumb.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="breadcrumb year_only">
+  <div class="bread_hover">
+    <div class="bread_links">
+      <%= link_to @year, send(path_calculation_method, @year), id: @year, 'aria-haspopup' => 'true', 'aria-label' => 'Escoge el año de los datos', 'aria-owns' => 'popup-year', 'aria-controls' => 'popup-year', 'aria-expanded' => 'false' %>
+      <i class="fa fa-sort-down"></i>
+    </div>
+
+    <div id="popup-year" class="line_browser clearfix" role="group">
+      <div class="bread_links clearfix">
+        <%= link_to @year, send(path_calculation_method, @year) %>
+        <i class="fa fa-sort-down"></i>
+      </div>
+
+      <div class="col" data-level="0">
+        <table  class="med_bg">
+          <% current_site.indicators.where(name: params[:action]).each do |indicator_date| %>
```

Comment:
Try to avoid using `params` in the views, that's a security issue. Use an instance variable with the indicator name.